analysers/analyser_merge_tourism_offices_FR.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Noémie Lehuby 2020                                         ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+
+class Analyser_Merge_Datatourisme_tourism_office_FR(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+        self.missing_official = self.def_class(item = 8010, id = 310, level = 3, tags = ['merge'],
+            title = T_('Tourism office not integrated'))
+
+        self.init(
+            u"https://www.data.gouv.fr/fr/datasets/datatourisme-la-base-nationale-des-donnees-du-tourisme-en-open-data/",
+            u"DATAtourisme, la base nationale des données du tourisme en Open Data",
+            CSV(Source(attribution = u"data.gouv.fr:DATAtourisme", millesime = "05/2020",
+                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/b9aa0996-4bdf-46d0-a375-cf7adc02d19b"), #TODO: no stable url ?
```

Comment:
u string no more required in py3.
```suggestion
            "https://www.data.gouv.fr/fr/datasets/datatourisme-la-base-nationale-des-donnees-du-tourisme-en-open-data/",
            "DATAtourisme, la base nationale des données du tourisme en Open Data",
            CSV(Source(attribution = "data.gouv.fr:DATAtourisme", millesime = "05/2020",
                    fileUrl = "https://www.data.gouv.fr/fr/datasets/r/b9aa0996-4bdf-46d0-a375-cf7adc02d19b"), #TODO: no stable url ?
```